iOS/Source/SunView.swift

```diff
@@ -155,7 +189,33 @@ class SunView: UIView {
         self.sun.tintColor = textColor
         self.moon.backgroundColor = backgroundColor
 
-        self.sunViewLocation = self.location(for: CGFloat(percentageInDay))
         self.sunPhase = sunPhase
+
+        if self.sunPhase == .sunrise || self.sunPhase == .solarNoon || self.sunPhase == .sunset {
+        self.sun.alpha = 1
```

Comment:
The indentation of the body of the if statement is off. Could you run SwiftFormat on this PR, please?